crates/ruff_linter/src/rules/flake8_pytest_style/rules/parametrize.rs

```diff
@@ -678,20 +678,82 @@ fn check_duplicates(checker: &mut Checker, values: &Expr) {
     }
 }
 
-fn handle_single_name(checker: &mut Checker, expr: &Expr, value: &Expr) {
+/// Generate [`Edit`]s to unpack single-element lists or tuples in the given [`Expr`].
+/// For instance, `[(1,) (2,)]` will be transformed into `[1, 2]`.
+fn unpack_single_element_items(checker: &Checker, expr: &Expr) -> Vec<Edit> {
```

Comment:
Nit: I suggest moving this method after `handle_single_name`. It eases the reading flow. I start with `check_names`, then jump to `handle_single_name`, and then keep reading this method.

Reply:
sure